src/api/application-api.md

```diff
@@ -181,7 +181,7 @@ Apart from `el`, you should treat these arguments as read-only and never modify
 - **Arguments:**
 
   - `{Element | string} rootContainer`
-  - `{boolean} isHydrate`
+  - `{boolean} [isHydrate]`
```

Comment:
**suggestion**: could we replace square brackets here with the note about optional component? Like we have on Directives

```
**Argument:** `attrOrProp (optional)`
```

Reply:
That would be inconsistent with the other optional parameters on this page, so I have created #442 to address that.

If you want to proceed with the `(optional)` format then this PR can be closed in deference to #442.